src/main/java/seedu/bankwithus/AccountList.java

```diff
@@ -46,5 +46,16 @@ public String getAllAccountDetails() {
         return temp;
     }
 
+    public Account getCurrentAccount() {
+        return accounts.get(0);
+    }
 
+    public void depositMoney(String depositAmountString) throws NumberFormatException, NullPointerException {
```

Comment:
Funcyion never throws any exception since it is all in  the try catch block

Reply:
I suggest to let it just throw the exception, and let the parser handle the exception instead i.e. remove the try catch block here and put it in the parser class. I think you can also remove the ui object if you do this